README.md

```diff
@@ -2,6 +2,53 @@
 
 A plugin for [Nautobot](https://github.com/nautobot/nautobot) [Chatops Plugin](https://github.com/nautobot/nautobot-plugin-chatops/)
 
+## Summary
+
+The Nautobot Chatops Ansible plugin extends the capabilities of the Nautobot Chatops plugin to include a new command available to the plugin. This is done by registering to the Python entry point in Nautobot Plugin Chatops, that provides functionality to the code written to interact with Ansible. This plugin introduces the following sub commands to the `ansible` command:
+
+* get_dashboard
+* get_inventory
+* get_projects
+* get_job_templates
+* run_job_template
+* get_jobs
+
+### Dashboard
+
+The dashboard command provides an Ansible Tower/AWX status dashboard. This gives a summary of:
+* How many hosts are on the host
+* How many failed hosts (last job is failed)
+* Total number of inventories
+* How many inventories have failed
+* How many project sync failures have occurred
+
+### Inventory
+
+The inventory sub-command provides for having inventory details. Providing 
+
+### Projects
```

Comment:
```suggestion
### Get Projects
```
Since all of the other sub headings are the command name, I thought it'd make sense to format this the same way.